### DISCUSSION TITLE
能否不依赖 @ExcelProperty 注解直接从 JSON 生成 Excel 呢

### DISCUSSION THREAD

**Discussion by NigelWu95** [alibaba/easyexcel#3261] [General]
能否？

**Comment by LSL1618** [comment]
完全不看文档的吗？[不创建对象的写](https://easyexcel.opensource.alibaba.com/docs/current/quickstart/write#%E4%B8%8D%E5%88%9B%E5%BB%BA%E5%AF%B9%E8%B1%A1%E7%9A%84%E5%86%99)

**Comment by LSL1618** [comment]
表头转成`List<List<String>>`，表内容转成`List<List<Object>>`。
